website/docs/reference/resource-configs/alias.md

```diff
@@ -4,10 +4,47 @@ description: "Read this guide to understand the alias configuration in dbt."
 datatype: string
 ---
 
-:::caution Heads up!
-This is a work in progress document. While this configuration applies to multiple resource types, the documentation has only been written for seeds.
 
-:::
+<Tabs>
+<TabItem value="model" label="Model">
+
+Specify a custom alias for a model in your `dbt_project.yml` file. 
+
+For example, if you have a model that calculates `sales_metric`s and want to give it a more user-friendly alias, you can alias it like this:
+
+<File name='dbt_project.yml'>
+
+```yml
+models:
+  your_project:
+    sales_metrics:
+      +alias: sales_dashboard
+```
+</File>
+
+This would return `analytics.finance.sales_dashboard` in the database, instead of the default `analytics.finance.sales_metrics`.
+</TabItem>
+
+<TabItem value="seeds" label="Seeds">
+
+Configure a seed's alias in your `dbt_project.yml` file. 
+
+For example, if you have a seed that represents `product_categories` and want to alias it as `categories_data`, you would alias like this:
+
+<File name='dbt_project.yml'>
+
+```yml
+seeds:
+  your_project:
+    product_categories:
+      +alias: categories_data
+```
+
+This would return the name `analytics.finance.categories_data` in the database.
+
+</File>
+</TabItem>
+</Tabs>
```

Comment:
It looks like snapshots and tests can also be aliased: 
- [Snapshots](https://github.com/dbt-labs/dbt-core/issues/4469) example
- Tests example (the longer-term options in [this comment](https://github.com/dbt-labs/dbt-core/issues/3626#issuecomment-886694083))